### PR TITLE
Refactor/orders page

### LINKED ISSUE #246: User's standing orders page (responsive)
Create a mobile friendly version of the orders. Ideally we would have the mocks for this.

### PR DESCRIPTION
**Pull request by W3stside**
Refactors orders page

closes #220 & closes #246

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr399--dexreact.review.gnosisdev.com

File: src/components/OrdersWidget/OrderRow.tsx [hunk 1]
```diff
@@ -66,13 +198,9 @@ const OrderRowWrapper = styled.div`
   }
 `
 
-const PendingLink: React.FC<Pick<Props, 'pending' | 'transactionHash'>> = ({ pending, transactionHash }) => {
-  if (!pending) {
-    return null
-  }
-
+const PendingLink: React.FC<Pick<Props, 'pending' | 'transactionHash'>> = ({ transactionHash }) => {
```
Comment (alfetopito) [inline src/components/OrdersWidget/OrderRow.tsx]: You can remove `pending` from props

File: src/components/OrdersWidget/OrderRow.tsx [hunk 2]
```diff
@@ -264,18 +443,23 @@ const OrderRow: React.FC<Props> = props => {
...
-            disabled={disabled}
-          />
+        <OrderRowWrapper className={'orderRow' + (pending ? ' pending' : '')} $open={openCard}>
+          {pending ? (
+            <PendingLink pending={pending} transactionHash={transactionHash} />
+          ) : (
+            <DeleteOrder
+              isMarkedForDeletion={isMarkedForDeletion}
+              toggleMarkedForDeletion={toggleMarkedForDeletion}
+              pending={pending}
```
Comment (alfetopito) [inline src/components/OrdersWidget/OrderRow.tsx]: Since this component won't be rendered when `pending` you can also remove from its props

Comment (W3stside) [reply src/components/OrdersWidget/OrderRow.tsx]: :+1:

File: src/components/OrdersWidget/OrderRow.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/OrdersWidget/OrderRow.tsx]: :+1:

**Pull request merged**